IFC4x3/Properties/t/TransverseSensitivityRatio_1t8VvJu1T3TPoInQI3Loie/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the transverse sensitivity ratio of the Earthquake Accelerometer.
```

Comment:
Generalize?

Reply:
Generalized the description: https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/1e8b0740cc2e7e000d58ef2f240b7e5a49640ee9